ZAPD/ZLimb.cpp

```diff
@@ -446,6 +446,7 @@ void ZLimb::ParseRawData()
 	{
 	case ZLimbType::LOD:
 		dList2Ptr = BitConverter::ToUInt32BE(rawData, rawDataIndex + 12);
+		[[fallthrough]];
 		// Intended fallthrough
```

Comment:
That's interesting. You can remove the comment now.
```suggestion
		[[fallthrough]];
```